Tribler/Core/Modules/restapi/util.py

```diff
@@ -4,9 +4,13 @@
 from twisted.web import http
 
 import Tribler.Core.Utilities.json_util as json
-from Tribler.Core.Modules.MetadataStore.serialization import time2float
+from Tribler.Core.Modules.MetadataStore.serialization import time2float, CHANNEL_TORRENT, float2time
 from Tribler.Core.Modules.restapi import VOTE_SUBSCRIBE
+from TriblerGUI.widgets.torrentdetailstabwidget import HEALTH_UNCHECKED, HEALTH_GOOD
```

Comment:
We should not import GUI widgets in the Core since we want to keep Core and GUI as independent as possible

Reply:
But we need both GUI and core to agree on a common set of symbols for communication.
So, either the GUI should depend on the core sources, or the core should depend on GUI sources,
or both should depend on some 3rd defs file/project.
What would you prefer?